modules/subworkflow/sv_wf.nf

```diff
@@ -29,10 +31,11 @@ workflow sv_wf
 
     // Put manta output and delly output into the same channel so they can be processed together in the group key
     // that they came in with i.e. (`idTumor`, `idNormal`, and `target`)
+  SomaticDellyCombine(
     SomaticDellyCall.out.dellyFilter4Combine
-    	.groupTuple(by: [0,1,2], size: 5)
-	.combine(manta4Combine, by: [0,1,2])
-	.set{ dellyMantaCombineChannel }
+      .groupTuple( by: [0,1,2], size: 5 )
```

Comment:
do you need to do `sort: true` same as `germline_sv_wf.nf`? I'm not sure but just pointing it out just in case.

Reply:
no, i will remove the sort parameter from germline. originally added that to both as i thought without the sort it would invalidate the cache, but it actually does not matter.

Reply:
germline now matches somatic.